cli/src/semgrep/commands/ci.py

```diff
@@ -592,6 +580,19 @@ def ci(
                 f"    https://semgrep.dev/orgs/{scan_handler.deployment_name}/supply-chain"
             )
 
+    output_handler.output(
```

Comment:
Are we sure that moving this doesn't affect anything else?

Reply:
when i was reading through the source, the `report_findings(..)`, doesn't alter any vars that we need here, so it should be fine.